Code/Framework/AzCore/AzCore/Debug/Profiler.h

```diff
@@ -318,6 +58,40 @@ namespace AZStd
 
 namespace AZ
 {
+    class ProfileScope
+    {
+    public:
+        static uint32_t GetSystemID(const char* system);
+
+        template<typename... T>
+        static void BeginRegion([[maybe_unused]] const char* system, [[maybe_unused]] char const* eventName, [[maybe_unused]] T const&... args)
+        {
+            // TODO: Verification that the supplied system name corresponds to a known budget
```

Comment:
Would be nice to add a Jira/GitHubIssue number so this doesnt get lost somehow.

Reply:
Yup it's there :)